definitions/scenarios/backup.rb

```diff
@@ -149,8 +140,8 @@ class BackupRescueCleanup < ForemanMaintain::Scenario
     end
 
     def compose
-      add_step_with_context(Procedures::Service::Start) if strategy != :online
-      add_steps_with_context(find_procedures(:maintenance_mode_off)) if strategy != :online
+      add_step_with_context(Procedures::Service::Start) if strategy == :offline
+      add_steps_with_context(find_procedures(:maintenance_mode_off)) if strategy == :offline
```

Comment:
Often I'm not a fan of guard clauses. Here they repeat so while you're at it, is this better?
```suggestion
      if strategy == :offline
        add_step_with_context(Procedures::Service::Start)
        add_steps_with_context(find_procedures(:maintenance_mode_off))
      end
```